.rubocop.yml

```diff
@@ -102,6 +102,10 @@ Style/DotPosition:
   - leading
   - trailing
 
+Style/IfUnlessModifier:
```

Comment:
Oh, this isn't anything to do with max-line-length, I just wanted to let you do your multiline if blocks without rubocop complaining